lib/addStyles.js

```diff
@@ -102,14 +102,13 @@ function addStylesToDom (styles, options) {
 		if(domStyle) {
 			domStyle.refs++;
 
-			for(var j = 0; j < domStyle.parts.length && item.parts.length; j++) {
```

Comment:
@michael-ciniawsky let's wait when we have test repo, then add tests and merge and publish. We have big problems with the tests (in many `webpack-contrib` repos), if we continue to ignore this, it will be very bad